phpcs.xml

```diff
@@ -2,13 +2,22 @@
 <ruleset name="PHP_CodeSniffer">
```

Comment:
Again, is there a reason to use both phpcs and php-cs-fixer?

Seems that php-cs-fixer does all of these checks/fixes no?

Reply:
file removed